green_mood_tracker/training_data.py

```diff
@@ -23,7 +23,12 @@ def get_raw_data(path = 'raw_data/', binary=False):
     sentiment140_final = sentiment140[['id','text','polarity']]
     sentiment140_final['source'] = 'sentiment140'
 
-    complete_data = pd.concat([sts_gold_final,kaggle_sentiment_train_final,kaggle_sentiment_test_final,sentiment140_final])
+    twitter_corpus = pd.read_csv('raw_data/twitter_corpus.csv',error_bad_lines=False)
```

Comment:
the syntax has changed to work with notebook and colab, so you just have to add path + 'twitter_corpus'